### PR TITLE
Update frontend/src/components/VSkeleton/VGridSkeleton.vue

### PR DESCRIPTION
**Pull request by viktoriussuwandi**
Fixes #[2440](https://github.com/WordPress/openverse/issues/2440) by @obulat

## Description
Hi [obulat](https://github.com/obulat),
just trying to help on this issue, please check it out

[Update frontend/src/components/VSkeleton/VGridSkeleton.vue](https://github.com/WordPress/openverse/commit/e3a9743be94d5a61129617ba080defcd1b0f4788)

## Checklist

- [X] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [X] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [X] My commit messages follow [best practices][best_practices].
- [X] My code follows the established code style of the repository.
- [X] I added or updated tests for the changes I made (if applicable).
- [X] I added or updated documentation (if applicable).
- [X] I tried running the project locally and verified that there are no visible errors.
- [X] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by obulat — CHANGES_REQUESTED** [review]
The tailwind classes need to have `@apply` to work inside the Vue component `<style>`.

File: frontend/src/components/VSkeleton/VGridSkeleton.vue [hunk 1]
```diff
@@ -63,14 +63,14 @@ export default defineComponent({
 
 <style scoped>
 .square {
-  aspect-ratio: 1 / 1;
+  aspect-square;
```
Comment (obulat) [inline frontend/src/components/VSkeleton/VGridSkeleton.vue:66]:
```suggestion
  @apply aspect-square;
```

File: frontend/src/components/VSkeleton/VGridSkeleton.vue [hunk 2]
```diff
@@ -63,14 +63,14 @@ export default defineComponent({
...
 }
 
-.masonry {
-  column-count: 2;
-  column-gap: 1rem;
+@screen md {
+  columns-1;
+  gap-x-4;
```
Comment (obulat) [inline frontend/src/components/VSkeleton/VGridSkeleton.vue:71]:
```suggestion
  @apply columns-1 gap-x-4;
```

File: frontend/src/components/VSkeleton/VGridSkeleton.vue [hunk 3]
```diff
@@ -63,14 +63,13 @@ export default defineComponent({
...
+  @apply aspect-square;
 }
 
 .masonry {
-  column-count: 2;
-  column-gap: 1rem;
+  @apply columns-2 gap-x-4 md:columns-3 lg:columns-5;
 
-  @screen md {
+  .masonry {
```
Comment (obulat) [inline frontend/src/components/VSkeleton/VGridSkeleton.vue:72]:
```suggestion
```

We already set the column count to 3 in the `.masonry` declaration in line 70 (`md:columns-3`), so lines 72 and 73 should be deleted.

File: frontend/src/components/VSkeleton/VGridSkeleton.vue [hunk 4]
```diff
@@ -63,14 +63,12 @@ export default defineComponent({
...
+  @apply aspect-square;
 }
 
 .masonry {
-  column-count: 2;
-  column-gap: 1rem;
+  @apply columns-2 gap-x-4 md:columns-3 lg:columns-5;
 
-  @screen md {
     column-count: 3;
```
Comment (obulat) [inline frontend/src/components/VSkeleton/VGridSkeleton.vue:72]: Line 72 also needs to be deleted.

**Review by obulat — CHANGES_REQUESTED** [review]
Sorry I haven't noticed the other style declarations below. You will also need to delete 
```
  @screen lg {
    column-count: 5;
}
```
The easiest and cleanest way would be to delete lines 71-76. This will remove extra blank lines, too.

**Review by obulat — APPROVED** [review]
There was one more extra bracket that was causing errors. I removed it, and now this PR is ready to go 🚀 
Thank you for your contribution, @viktoriussuwandi!

**Review by AetherUnbound — APPROVED** [review]
Changes LGTM! Thanks for your contribution!

**Pull request merged**